main.go

```diff
@@ -48,8 +48,15 @@ func rssHandler(sourceFeeds []sourceFeed) http.Handler {
 
 		sort.Sort(byCreated(master.Items))
 
-		result, _ := master.ToAtom()
-		fmt.Fprintln(rw, result)
+		result, err := master.ToAtom()
+		if err != nil {
+			log.Fatal(err)
```

Comment:
Should the HTTP status code change in this branch to say Internal Server Error? We are still flowing through to `fmt.Fprintln(rw, result)` below... which will respond with a 200 OK?